packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -13,10 +13,13 @@
 // limitations under the License.
 //
 
+/* eslint-disable @typescript-eslint/no-var-requires */
+
 import myProbotApp from '../src/snippet-bot';
 
 import {resolve} from 'path';
 import {Probot} from 'probot';
+import snapshot from 'snap-shot-it';
 import nock from 'nock';
 import * as fs from 'fs';
 import {expect} from 'chai';
```

Comment:
Let's remove `chai` here, and instead use `import assert from 'assert'`.  `assert` is baked into node.js, and doesn't require a third party dependency.

Reply:
done